### DISCUSSION TITLE
Should be clearer that discussion search supports same filters as issue search (and ignore whitespace)

### DISCUSSION THREAD

**Discussion by jugglingcats** [community/community#49] [Discussions] [answered]
I tried using search using `author: jugglingcats` (note the whitespace) and got no results. Is there a reason whitespace isn't trimmed?

I realised this did in fact work without the space and I could also use `commenter` and `mentions`. I think this should be clearer to people.

Tx

**Comment by becca** [comment, marked as answer]
👋 you are correct that the spacing is not supported in our search - both in Discussions, notifications and globally. we most certainly can make it easier to know all the options for search (including the two you mentioned - i.e. `commenter`)

thank you for this feedback!

**Comment by JDLH** [comment]
+1. I keep typing a space character between `author:` and UserName. The response is a search result of zero messages. It would be best if the parser would forgive my mistake of typing a space character. An alternative helpful response would be for the search feature to give an error message when it detects `author:` followed by Null String. That combination can't be meaningful, so an error message would be more helpful than just an empty result.
